src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -904,4 +944,229 @@ std::string primitive_inst::get_implementation_name() const {
+        ob << can_share_buffer();
+        ob << is_constant();
+
+        ob << _outputs[0]->get_layout();
+        const auto _allocation_type = _outputs[0]->get_allocation_type();
+        ob << make_data(&_allocation_type, sizeof(_allocation_type));
+
+        ob << _node->get_memory_dependencies();
+
+        ob << _deps.size();
+        for (const auto& dep : _deps) {
+            ob << dep->id();
+        }
+
+        ob << _exec_deps.size();
+        for (const auto& dep : _exec_deps) {
+            ob << dep->id();
+        }
+
+        ob << _intermediates_memory.size();
+        for (const auto& ibuf : _intermediates_memory) {
+            ob << ibuf->get_layout();
+            const auto _allocation_type = ibuf->get_allocation_type();
+            ob << make_data(&_allocation_type, sizeof(_allocation_type));
+        }
+    }
+}
+
+void primitive_inst::convert_args(const kernel_arguments_data& args, kernel_arguments_data_idx& args_idx) const {
+    if (args.inputs.size() > 0) {
+        args_idx.inputs.resize(args.inputs.size());
+        for (uint32_t idx = 0; idx < args.inputs.size(); ++idx) {
+            args_idx.inputs[idx] = get_index_in_deps(args.inputs[idx]);
+        }
+    }
+
+    args_idx.weights = (args.weights == nullptr) ? -1 : get_index_in_deps(args.weights);
+    args_idx.recurrent = (args.recurrent == nullptr) ? -1 : get_index_in_deps(args.recurrent);
+    args_idx.hidden = (args.hidden == nullptr) ? -1 : get_index_in_deps(args.hidden);
+    args_idx.cell = (args.cell == nullptr) ? -1 : get_index_in_deps(args.cell);
+    args_idx.bias = (args.bias == nullptr) ? -1 : get_index_in_deps(args.bias);
+    args_idx.weights_zero_points = (args.weights_zero_points == nullptr) ? -1 : get_index_in_deps(args.weights_zero_points);
+    args_idx.activations_zero_points = (args.activations_zero_points == nullptr) ? -1 : get_index_in_deps(args.activations_zero_points);
+    args_idx.compensation = (args.compensation == nullptr) ? -1 : get_index_in_deps(args.compensation);
+    args_idx.lookup_table = (args.lookup_table == nullptr) ? -1 : get_index_in_deps(args.lookup_table);
+    args_idx.scale_table = (args.scale_table == nullptr) ? -1 : get_index_in_deps(args.scale_table);
+    args_idx.slope = (args.slope == nullptr) ? -1 : get_index_in_deps(args.slope);
+
+    if (args.fused_op_inputs.size() > 0) {
+        args_idx.fused_op_inputs.resize(args.fused_op_inputs.size());
+        for (uint32_t idx = 0; idx < args.fused_op_inputs.size(); ++idx) {
+            args_idx.fused_op_inputs[idx] = get_index_in_deps(args.fused_op_inputs[idx]);
+        }
+    }
+
+    args_idx.split = args.split;
+}
+
+int32_t primitive_inst::get_index_in_deps(memory::cptr arg) const {
+    uint32_t idx = 0;
+
+    for (idx = 0; idx < _deps.size(); ++idx) {
+        if (arg == _deps[idx]->_outputs[0])
+            break;
+    }
+
+    if (idx == _deps.size())
+        std::cout << "[get_index_in_deps]: not found" << std::endl;
```

Comment:
Why? It should be either removed or changed to exception.

Reply:
https://github.com/openvinotoolkit/openvino/pull/13986/files#r1024716869
Changed to throw an exception. Thank you.